generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -36,6 +36,7 @@ quarkus.micrometer.export.prometheus.path=/management/jhimetrics
 
 quarkus.smallrye-health.root-path=/management/health
 
+<%if (authenticationType === 'jwt') { _%>
 mp.jwt.verify.publickey.location=META-INF/resources/publicKey.pem
```

Comment:
Do you have to create public/private keys for JWTs? If so, are these generated for each new app, or are they the same one for each app?